test/Npgsql.Tests/ReaderTests.cs

```diff
@@ -151,6 +151,11 @@ public async Task RecordsAffected()
             reader = await cmd.ExecuteReaderAsync(Behavior);
             reader.Close();
             Assert.That(reader.RecordsAffected, Is.EqualTo(4));
+
+            cmd = new NpgsqlCommand($"MERGE INTO {table} S USING (SELECT 2 as int) T ON T.int = S.int WHEN MATCHED THEN UPDATE SET int = S.int", conn);
```

Comment:
This is missing the PG15 check (see errors below)

Reply:
Yep thanks. That's the commit above.